qt5/build.sh

```diff
@@ -1,96 +1,93 @@
-                -nomake examples \
-                -nomake tests \
-                -verbose \
-                -no-libudev \
-                -gtkstyle \
-                -qt-xcb \
-                -qt-pcre \
-                -qt-xkbcommon \
-                -xkb-config-root $PREFIX/lib \
-                -dbus
+    # see https://doc.qt.io/qt-5/linux-requirements.html
+    ./configure \
+        -verbose \
+        -prefix $PREFIX \
+        -bindir $PREFIX/lib/qt5/bin \
+        -headerdir $PREFIX/include/qt5 \
+        -libdir $PREFIX/lib \
+        -archdatadir $PREFIX/lib/qt5 \
+        -datadir $PREFIX/share/qt5 \
+        -L $PREFIX/lib \
+        -R $PREFIX/lib \
+        -I $PREFIX/include \
+        -opensource \
+        -confirm-license \
+        -release \
+        -shared \
+        -nomake examples \
+        -nomake tests \
+        -skip enginio \
+        -qt-pcre \
+        -system-xkbcommon-x11 \
+        -no-gtkstyle \
+        -dbus \
+        -no-gstreamer
 fi
 
 if [ `uname` == Darwin ]; then
     # Let Qt set its own flags and vars
-    for x in OSX_ARCH CFLAGS CXXFLAGS LDFLAGS
-    do
+    for x in OSX_ARCH CFLAGS CXXFLAGS LDFLAGS; do
         unset $x
     done
 
     MACOSX_DEPLOYMENT_TARGET=10.7
     MAKE_JOBS=$(sysctl -n hw.ncpu)
 
-    ./configure -prefix $PREFIX \
-                -libdir $PREFIX/lib \
-                -bindir $PREFIX/lib/qt5/bin \
-                -headerdir $PREFIX/include/qt5 \
-                -archdatadir $PREFIX/lib/qt5 \
-                -datadir $PREFIX/share/qt5 \
-                -L $PREFIX/lib \
-                -I $PREFIX/include \
-                -release \
-                -opensource \
-                -confirm-license \
-                -shared \
-                -nomake examples \
-                -nomake tests \
-                -verbose \
-                -skip qtwebengine \
-                -qt-pcre \
-                -platform macx-g++ \
-                -no-c++11 \
-                -no-framework \
-                -no-dbus \
-                -no-mtdev \
-                -no-harfbuzz \
-                -no-xinput2 \
-                -no-xcb-xlib \
-                -no-libudev \
-                -no-egl
+    ./configure \
+        -platform macx-g++ \
+        -verbose \
+        -prefix $PREFIX \
+        -bindir $PREFIX/lib/qt5/bin \
+        -headerdir $PREFIX/include/qt5 \
+        -libdir $PREFIX/lib/qt5 \
+        -archdatadir $PREFIX/lib/qt5 \
+        -datadir $PREFIX/share/qt5 \
+        -L $PREFIX/lib \
+        -R $PREFIX/lib \
+        -I $PREFIX/include \
+        -opensource \
+        -confirm-license \
+        -release \
+        -shared \
+        -nomake examples \
+        -nomake tests \
+        -skip qtwebengine \
+        -qt-pcre \
+        -no-c++11 \
+        -no-framework \
+        -no-dbus \
+        -no-mtdev \
+        -no-harfbuzz \
+        -no-xinput2 \
+        -no-xcb-xlib \
+        -no-libudev \
+        -no-egl
```

Comment:
What options did you exactly changed and/or added here? Please leave them in the same order as before and your new ones at the end so I can easily what you've changed :-)